connector/servicegraphconnector/connector.go

```diff
@@ -406,6 +415,10 @@ func (p *serviceGraphConnector) aggregateMetricsForEdge(e *store.Edge) {
 	metricKey := p.buildMetricKey(e.ClientService, e.ServerService, string(e.ConnectionType), e.Dimensions)
 	dimensions := buildDimensions(e)
 
+	if p.config.VirtualNodeExtraLabel {
+		dimensions = addExtraLabel(dimensions, virtualNodeLabel, string(e.VirtualNodeLabel))
+	}
```

Comment:
I understand we only want to add this label if the edge has a virtual node, right? We should check that

Reply:
I think it can be confusing, but I don't have a strong opinion on it.

Reply:
OK then, I'll opt for leaving it with an empty value sometimes but always present. We can review this if it's an issue.